packages/example-app-base/src/components/App/App.tsx

```diff
@@ -20,13 +20,23 @@ const getClassNames = classNamesFunction<IAppStyleProps, IAppStyles>();
 export class AppBase extends React.Component<IAppProps, IAppState> {
   public state: IAppState = { isMenuVisible: false };
   private _classNames: IProcessedStyleSet<IAppStyles>;
+  private _baseUrl: string;
+
+  constructor(props: IAppProps) {
+    super(props);
+
+    const doc = getDocument();
+    this._baseUrl = doc ? document.location.href : '';
+  }
 
   public render(): JSX.Element {
     const { appDefinition, styles, responsiveMode = ResponsiveMode.xLarge, theme } = this.props;
     const { customizations } = appDefinition;
     const { isMenuVisible } = this.state;
 
-    const classNames = (this._classNames = getClassNames(styles, { responsiveMode, theme }));
+    const showOnlyExamples = this._baseUrl.indexOf('docsExample=true') > -1;
```

Comment:
Since this check is now done multiple places, it would be good to add a shared utility function, probably under `example-app-base/src/utilities/showOnlyExamples.ts`

Reply:
Done

Reply:
Thanks!